modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -346,12 +337,12 @@ settings which are further used for app or client authentication. Browser logon
 'token_auth_enforced' => false,
 ....
 
-=== Allows to specify additional login buttons on the logon screen for e.g. SSO integration.
-
-'login.alternatives' => [
-  ['href' => 'https://www.testshib.org/Shibboleth.sso/ProtectNetwork?target=https%3A%2F%2Fmy.owncloud.tld%2Flogin%2Fsso-saml%2F', 'name' => 'ProtectNetwork', 'img' => '/img/PN_sign-in.gif'],
-  ['href' => 'https://www.testshib.org/Shibboleth.sso/OpenIdP.org?target=https%3A%2F%2Fmy.owncloud.tld%2Flogin%2Fsso-saml%2F', 'name' => 'OpenIdP.org', 'img' => '/img/openidp.png'],
-]
+=== Defines additional login buttons on the logon screen
+Allows to specify additional login buttons on the logon screen for e.g. SSO integration
```

Comment:
The start of this sentence doesn't read correctly. I suggest starting with: "Allows for specifying additional..."